Framework/CMakeLists.txt

```diff
@@ -70,6 +74,9 @@ add_subdirectory (Types)
 include_directories (Kernel/inc)
 add_subdirectory (Kernel)
 
+include_directories (Parallel/inc)
+add_subdirectory (Parallel)
```

Comment:
There are (or rather will be) bits in `Parallel` that are available also in non-MPI builds, I think.